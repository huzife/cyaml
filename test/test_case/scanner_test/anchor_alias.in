---
a: &a 1
b: &b
  b1: 2
  b2: *a
*b : 3
