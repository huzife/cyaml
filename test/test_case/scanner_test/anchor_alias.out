(DOC_START)#
(BLOCK_MAP_START)#
(KEY)#
(SCALAR, a)#
(VALUE)#
(ANCHOR, a)#
(SCALAR, 1)#
(KEY)#
(SCALAR, b)#
(VALUE)#
(ANCHOR, b)#
(BLOCK_MAP_START)#
(KEY)#
(SCALAR, b1)#
(VALUE)#
(SCALAR, 2)#
(KEY)#
(SCALAR, b2)#
(VALUE)#
(ALIAS, a)#
(BLOCK_MAP_END)#
(KEY)#
(ALIAS, b)#
(VALUE)#
(SCALAR, 3)#
(BLOCK_MAP_END)#
